README.md

```diff
@@ -54,6 +55,37 @@ quoting mechanism.
 
 After the conversion, the bytes can be use in the field `metadata` when originating the contract.
 
+## Generating the contract storage
+
+You can generate a big_map metadata that can be used with `make out/storage_default.tz`
+and is TZIP-16 compliance.
+
+```bash
+make metadata_bigmap output_file=out/metadata_map
+```
+
+Then you can include it in storage generation command like below:
+
+```bash
+make out/storage_default.tz \
+    fee_bps=10 \
+    ctez_burn_fee_bps=5 \
+    x_token_id=0 \
+    y_token_id=0 \
+    x_token_address=KT1PWx2mnDueood7fEmfbBDKx1D9BAnnXitn \
+    y_token_address=KT1PWx2mnDueood7fEmfbBDKx1D9BAnnXitn \
+    x_token_symbol=x \
+    x_token_name="Token X" \
+    x_token_decimals=1 \
+    y_token_symbol=y \
+    y_token_name="Token Y" \
+    y_token_decimals=1 \
+    metadata_map="$(cat out/metadata_map)"
```

Comment:
This field should be mentioned in the `docs/compilation.md` file instead.

Reply:
Added to compilation.md.